File: Data Structures & Algorithms/missing-number/submission-1.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size() + 1;
        unordered_map<int,int>mp;
        for(int i = 0; i < nums.size(); i++){
            mp[nums[i]] = i;
        }
        for(int i = 0; i < n; i++){
            if(mp.count(i) == 0){
                return i;
            }
        }
        return 0;
        
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        frequency = {}
        for n in nums:
            frequency[n] = 1 + frequency.get(n, 0)

        heap = []

        for num in frequency.keys():
            heapq.heappush(heap, (frequency[num], num))

        while len(heap) > k:
            heapq.heappop(heap)
        
        myList = []
        for i in range(k):
            myList.append(heapq.heappop(heap)[1])

        return myList   